loop.js

```diff
@@ -0,0 +1,7 @@
+const loop = (currentValue, test, update, body) => {
+    if (test(currentValue)) {
+        body(currentValue)
```

Comment:
What does that `body` function do? Is it mutate the `currentValue`? Maybe add some comment here? 🤔